rdkit/Chem/Draw/UnitTestDraw.py

```diff
@@ -41,7 +42,7 @@
 class TestCase(unittest.TestCase):
   showAllImages = False
 
-  def test_interactive(self):
+  def atest_interactive(self):
```

Comment:
you disabled a bunch of tests in here by renaming them.
We can think about removing the tests of the old drawing code, but let's do that in separate PR if we go that way.
For this one please re-enable these tests.

Reply:
Arg!  I need a commit hook for this, I rename tests so I don’t go insane with logs :)